### DISCUSSION TITLE
System setup for the ML course

### DISCUSSION THREAD

**Discussion by cjd2003** [microsoft/ML-For-Beginners#252] [Q&A] [answered]
Hi,
In order to configure my system for this course, I tried clicking the link in the first lesson 1 - Into_to _ML but it took me to a set of videos for "Introduction to Python".. I was wondering if I could get the correct link. 

Thank you.

**Comment by jlooper** [comment, marked as answer]
hi, you should start here: https://github.com/microsoft/ML-For-Beginners/blob/main/1-Introduction/1-intro-to-ML/README.md

Use the table's column on the right to navigate to each lesson. If there's a misconfigured link, however, please tell me which file and I'll fix it. thanks!

**Reply by cjd2003** [reply]
Hi, 
I am little confused.. was the link (circled in the screenshot below) supposed to take me to a YouTube playlist for 'Python for Beginners'? or to some other resources to setup my machine?

This link is found in "1-Introduction/1-intro-to-ML/README.md"
<img width="1050" alt="Screen Shot 2021-08-06 at 12 41 05 PM" src="https://user-images.githubusercontent.com/59383518/128563569-ba0e73d9-d0cd-414b-9f96-c66a7a056ec7.png">

**Comment by alfredodeza** [comment]
@cjd2003 I think you are right. I went through the videos in the playlist and found two that might be useful. I'll get the README updated to use those to specifically instead of linking to the whole set of Python programming videos.
